README.md

```diff
@@ -50,25 +50,6 @@ server-url = https://www.dockstore.org/api
 
 1. Ensure that you are using Java 11. Java 8 (both Open and Oracle) will not work.
 
-### Migration to Dockstore 1.3
```

Comment:
Surprised this changed in the hotfix, but seems fine.

Reply:
This was me, testing admin push.